packages/coding-components/templates/angular/src/index.html

```diff
@@ -7,6 +7,7 @@
   <base href="/" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
+  <link rel="stylesheet" href="https://js.arcgis.com/4.29/esri/themes/light/main.css" />
```

Comment:
Let's switch this to the ESM CDN and it should moved from index.html and into the component directory's .css file.